docs/externalGrading.md

```diff
@@ -163,15 +163,17 @@ The following is an example of a well-structured course layout:
 
 ## Grading results
 
-Your grading process must write its results to `/grade/results/results.json`. The result only has 2 mandatory fields: `succeeded` and `score`. `succeeded` indicates if the tests were able to run successfully, or if they failed due to some error. `score` is the score, and should be a floating point number in the range [0.0, 1.0].
+Your grading process must write its results to `/grade/results/results.json`. The result only has one mandatory field: `score`, which is the score for the submitted attempt, and should be a floating point number in the range [0.0, 1.0].
```

Comment:
If `gradable = false` then should we make the `score` optional? It seems weird to be forced to set a score when it's going to be ignored.